cmd/utils.go

```diff
@@ -104,33 +104,31 @@ func GetFileDetails(filePath string) (modFileContent []byte, modFileStat os.File
 	return
 }
 
-func outputToMap(output string) map[string]bool {
+func listToMap(output string) map[string]bool {
 	lineOutput := strings.Split(output, "\n")
 	mapOfDeps := map[string]bool{}
-
 	for _, line := range lineOutput {
-		splitLine := strings.Split(line, " ")
-		lineLen := len(splitLine)
-		if lineLen == 2 {
-			mapOfDeps[splitLine[0]+"@"+splitLine[1]] = true
+		//expected syntax : github.com/name@v1.2.3
```

Comment:
//expected syntax : github.com/name@v1.2.3
-->
// The expected syntax is: github.com/name@v1.2.3

The same goes for the comment in the ```graphToMap``` function.